app/kubernetes/converters/application.js

```diff
@@ -353,13 +357,22 @@ class KubernetesApplicationConverter {
         (claims.length === 0 || (claims.length > 0 && formValues.DataAccessPolicy === KubernetesApplicationDataAccessPolicies.SHARED && rwx))) ||
       formValues.ApplicationType === KubernetesApplicationTypes.DAEMONSET;
 
+    const pod = formValues.ApplicationType === KubernetesApplicationTypes.POD;
+
+    let serviceSelector = {};
```

Comment:
I believe this can be moved to the pod converter. this will make it clear that this code is only related to pod